src/formats/jsonLd.ts

```diff
@@ -74,56 +48,31 @@ export const getJsonLdParser = (): Parser => {
     onComplete: (callback) => {
       onCompleteCallbacks.push(callback);
     },
-    parse: async (source, resourceInfo) => {
-      const jsonld = (await import("jsonld")).default;
+    parse: async (source, resourceInfo) =>
+      new Promise((res) => {
```

Comment:
```suggestion
parse: (source, resourceInfo) => {
      // eslint-disable-next-line no-void
      void new Promise<void>((res) => {
```
`parse` should be a void function, so not return a Promise. Feel free to add an appropriate comment on the linting exception.

Reply:
As per our offline conversation, this will not be changed in the current version, and will only be implemented later as it could be a breaking change for developers relying on undocumented behavior.